index.d.ts

```diff
@@ -1,71 +1,257 @@
-export type DataType = 'string';
+export interface WikidataResponse {
+	entities: {
+		[id: string]: Item;
+	};
+}
+
+export interface Item {
+	pageid: number;
+	ns: number;
+	title: string;
+	lastrevid: number;
+	modified: string;
+	type: string;
```

Comment:
I think that type can be "item" and that other (jet to be added) types can have an other uniqe type